api_root_lambda/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:12-alpine
```

Comment:
This file is not used but would be the configuration for building this app in docker.